backtester/config/README.md

```diff
@@ -127,6 +127,7 @@ See below for a set of tables and fields, expected values and what they can do
 | APISecretOverride | Will set the GoCryptoTrader exchange to use the following API Secret | `5678` |
 | APIClientIDOverride | Will set the GoCryptoTrader exchange to use the following API Client ID | `9012` |
 | API2FAOverride | Will set the GoCryptoTrader exchange to use the following 2FA seed | `hello-moto` |
+| APISubaccountOverride | Will set the GoCryptoTrader exchange to use the following subaccount on supported exchanges | `subzero` |
```

Comment:
Do you think there could be any additional place to put this? Our configs for FTX and config look a little outdated, so I'm not asking you to rewrite them, but I think it would be good to highlight this somewhere for the main GCT that FTX can use a subaccount

Reply:
Yep, but this will be done in another PR since there's quite a few docs outdated related to config settings. Will also be better to advertise this feature more once it is linked to more subsystems (portfolio account balances based on subaccounts etc.) as this was just a basic add-on for this PR.